packages/components/card.css

```diff
@@ -0,0 +1,180 @@
+.card {
+    margin: 0.625rem;
```

Comment:
for these small values use px instead of rem

Reply:
if its within 0.5rem we can keep that in px , i think 0.625rem is not problem